### PR TITLE
Feature/contact us

### PR DESCRIPTION
**Pull request by senuravihanjayadeva**

### CODE REVIEW

File: src/modules/ContactUsModules/ContactForm/ContactForm.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import 'react-notifications/lib/notifications.css';
+import { NotificationContainer, NotificationManager } from 'react-notifications';
+import './ContactForm.css';
+import emailjs from 'emailjs-com';
+import ContactUsImg from '../../../assets/contactus/contactus.svg';
+
+function sendEmail(e) {
```
Comment (gihanrcg) [inline src/modules/ContactUsModules/ContactForm/ContactForm.jsx:8]: Can we use an arrow function here ?

File: src/routes/AppRoutes.jsx [hunk 2]
```diff
@@ -12,6 +13,7 @@ const AppRoutes = () => (
         <Route path="/" component={Home} exact />
         <Route path="/about" component={AboutUs} exact />
         <Route path="/events" component={Events} exact />
+        <Route path="/contactus" component={ContactUs} exact />
```
Comment (gihanrcg) [inline src/routes/AppRoutes.jsx:16]: let's change pasth here to 'contact'

Comment (gihanrcg) [inline src/routes/AppRoutes.jsx:16]:
```suggestion
        <Route path="/contact" component={ContactUs} exact />
```

File: .eslintrc [hunk 3]
```diff
@@ -125,6 +125,18 @@
         // "jsdoc/require-param-type": "warn",
         // "jsdoc/require-returns-description": "warn",
         // "jsdoc/require-returns-type": "warn",
-      
-      }
+        "jsx-a11y/label-has-associated-control": ["error", {
```
Comment (gihanrcg) [inline .eslintrc:128]: What is happening here ?

**Review by gihanrcg — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 3 above)
Comment (senuravihanjayadeva) [reply .eslintrc:128]: I got the following error
"error  Form label must have associated control"

Comment (gihanrcg) [reply .eslintrc:128]: for every label should have a control
`labelFor`

Comment (gihanrcg) [reply .eslintrc:128]: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/718

Comment (gihanrcg) [reply .eslintrc:128]: But it is not a big deal. We can ignore this

**Review by gihanrcg — APPROVED** [review]

**Pull request merged**